ForumProject/communications/static/room.js

```diff
@@ -0,0 +1,115 @@
+console.log("Sanity check from room.js.");
+
+const roomName = JSON.parse(document.getElementById('roomName').textContent);
+
+let chatLog = document.querySelector("#chatLog");
+let chatMessageInput = document.querySelector("#chatMessageInput");
+let chatMessageSend = document.querySelector("#chatMessageSend");
+let onlineUsersSelector = document.querySelector("#onlineUsersSelector");
+
+// adds a new option to 'onlineUsersSelector'
+function onlineUsersSelectorAdd(value) {
+    if (document.querySelector("option[value='" + value + "']")) return;
+    let newOption = document.createElement("option");
+    newOption.value = value;
+    newOption.innerHTML = value;
+    onlineUsersSelector.appendChild(newOption);
+}
+
+// removes an option from 'onlineUsersSelector'
+function onlineUsersSelectorRemove(value) {
+    let oldOption = document.querySelector("option[value='" + value + "']");
+    if (oldOption !== null) oldOption.remove();
+}
+
+// focus 'chatMessageInput' when user opens the page
+chatMessageInput.focus();
+
+const ENTER_KEY_CODE = 13;
+// submit if the user presses the enter key
+chatMessageInput.onkeyup = function(e) {
+    if (e.keyCode === ENTER_KEY_CODE) {  // enter key
+        chatMessageSend.click();
+    }
```

Comment:
Using the `keyCode` property is deprecated. It’s better to use` e.key` which is more readable